R/data.table.R

```diff
@@ -12,9 +12,13 @@ methods::setPackageName("data.table",.global)
 #   (2) export() in NAMESPACE
 #   (3) add to vignettes/datatable-importing.Rmd#globals section
 .SD = .N = .I = .GRP = .NGRP = .BY = .EACHI = NULL
+. = function(...) NULL
+J = function(...) NULL
+patterns = function(...) NULL
+measure = function(...) NULL
```

Comment:
I don't think patterns and measure should be here, since they are defined in fmelt.R (which I guess is evaluated after data.table.R, over-writing this definition? and that is why the code still works / tests still pass?)

Reply:
Yeah when i moved the masks of both measure and patterns to fmelt.R below the original functions,  a lot of test and 2 vignettes seems to cause issue, we might have to change a lot of tests and some code in vignette `datatable-reshape.Rmd, datatable-sd-usage.Rmd `, is that ok? and moving masks to fmelt.R is good enough or somewhere else will be better?

Reply:
no, just delete them, no mask needed since they are defined in fmelt.R

Reply:
done 👍